tsunamibayes/scenario.py

```diff
@@ -262,21 +273,21 @@ def sample(self,nsamples,output_dir=None,save_freq=1,verbose=False):
                             self.proposal_logpdf(proposal,self.samples.loc[i-1])
                 alpha = np.exp(alpha)
 
-            if verbose: print("alpha = {:.3E}".format(alpha))
+            if verbose: print("The acceptance probablity, alpha = {:.3E}".format(alpha))
 
             # prior, likelihood, and posterior logpdf values
             bayes_data = pd.Series([prior_logpdf,llh,prior_logpdf+llh],index=self.bayes_data_cols)
 
             # accept/reject
             accepted = (np.random.rand() < alpha)
             if accepted:
-                if verbose: print("Proposal accepted",flush=True)
+                if verbose: print("***Proposal accepted***",flush=True)
                 self.samples.loc[i] = proposal
                 self.model_params.loc[i] = model_params
                 self.model_output.loc[i] = model_output
                 self.bayes_data.loc[i] = bayes_data
             else:
-                if verbose: print("Proposal rejected",flush=True)
+                if verbose: print("***Proposal rejected***",flush=True)
```

Comment:
This was just to make this line a little more recognizable in the code as you scroll through.